composer.json

```diff
@@ -13,11 +13,12 @@
     "php": ">=7.4",
     "ext-json": "*",
     "ext-pdo": "*",
-    "keboola/common-exceptions": "^1.0",
-    "keboola/csv": "^2.1",
+    "keboola/common-exceptions": "^1.1",
+    "keboola/csv": "^2.2",
     "keboola/db-extractor-config": "^1.4",
     "keboola/db-extractor-ssh-tunnel": "^1.1",
     "keboola/db-extractor-table-format": "^3.0",
+    "keboola/db-extractor-adapter": "dev-webrouse-COM-365-init",
```

Comment:
Po mergnuti druheho PR https://github.com/keboola/db-extractor-adapter/pull/1 sa to upravi.